regulatory/data/system.yml

```diff
@@ -0,0 +1,77 @@
+# The name of the project
+project_name: Ventilator Software
+
+# List optional extensions
+md_extensions:
+  - 'rdm.md_extensions.section_numbers.SectionNumberExtension'
+  - 'rdm.md_extensions.vocabulary_extension.VocabularyExtension'
+  - 'rdm.md_extensions.audit_notes.AuditNoteExclusionExtension'
+#  - 'rdm.md_extensions.audit_notes.AuditNoteInclusionExtension'
+# audit notes can be excluded or included by changing which of the above two lines are commented out.
+
+# Standard Version
+standard: IEC62304:2006
+
+# Is the software project part of a medical device with a hardware component,
+# or is the software a medical device by-itself?
+is_software_only_device: false
+
+# The location where the system requirements are stored; only used if
+# `is_software_only_device == false`
+system_requirements_location: SYSTEM REQUIREMENTS SOFTWARE
+
+# The project management tool
+project_management_tool: GitHub
+
+# The location where risk analysis outputs are stored
+risk_matrix_location: '`risk.yml`'
```

Comment:
Hm that file looks hand-written, is there some tool that is supposed to produce it based on an analysis of something else?